cscs-checks/tools/profiling_and_debugging/ipm_mpi_omp.py

```diff
@@ -5,7 +5,7 @@
 
 
 @rfm.required_version('>=2.14')
-@rfm.parameterized_test(['C'], ['C++'], ['F90'])
+@rfm.parameterized_test(['C++'], ['F90'])
```

Comment:
Don't you need C anymore?

Reply:
No, if it works in C++, it will work in C (+ it will take less time to run)